apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -81,7 +81,8 @@ public static async Task<BigtableClient> CreateAsync(ServiceEndpoint endpoint =
             Create(BigtableServiceApiClient.Create(GaxPreconditions.CheckNotNull(callInvoker, nameof(callInvoker)), settings));
 
         /// <summary>
-        /// Gets the value which specifies routing for replication. If null, the "default" application profile will be used by the server.
+        /// Gets the value which specifies routing for replication.
+        /// If null or emtpy, the "default" application profile will be used by the server.
```

Comment:
emtpy => empty

Reply:
(I'm not sure whether returning null is appropriate here, or throwing an exception like our other virtual members do. Any thoughts either way?)

Reply:
(Same comments in BigtableApiServiceClient)

Reply:
Fixed. Yeah perhaps throwing is better.